include/integratorxx/quadratures/womersley/womersley_98.hpp

```diff
@@ -0,0 +1,197 @@
+      -9.6423952683701175e-01, -2.5648985477379754e-01,
+      -6.6746455211133593e-02, -8.3498948067555834e-01,
+      -5.4759466504315746e-01, -5.4153947016201076e-02,
+      -5.9915581457647527e-01, -7.7095621553230831e-01,
+      -2.1596023613457457e-01, -2.0623913289015952e-01,
+      -9.1770198988596019e-01, -3.3954157009718855e-01,
+      1.7550358482310907e-01,  -9.4040258288569278e-01,
+      -2.9127559770113853e-01, 4.7844025267248980e-01,
+      -8.3269136985879699e-01, -2.7878308267426194e-01,
+      6.9275092819504880e-01,  -5.9877717013136134e-01,
+      -4.0194782250234568e-01, 8.1518267499384800e-01,
+      -1.7338238811797230e-01, -5.5264432855172163e-01,
+      7.3579508274743821e-01,  1.4062753447803539e-01,
+      -6.6244206746803125e-01, 5.3634037587822736e-01,
+      4.2185318097166796e-01,  -7.3101223991591757e-01,
+      3.2746484070912718e-01,  6.7829336457784051e-01,
+      -6.5778787589086707e-01, -1.9369444998744729e-02,
+      7.9032620596293868e-01,  -6.1238003949236230e-01,
+      -3.5046513262664786e-01, 7.5857214939479445e-01,
+      -5.4931091831088485e-01, -6.1717221716676196e-01,
+      5.5915663712140562e-01,  -5.5357231643259053e-01,
+      -7.6726068622646537e-01, 1.6240661135912177e-01,
+      -6.2043140793978235e-01, -8.7346825656259486e-01,
+      -1.9402005866523162e-01, -4.4655282063048379e-01,
+      -7.4936073836579620e-01, -4.9467203078821015e-01,
+      -4.4017958352442554e-01, -4.2931701893592167e-01,
+      -7.3301067449606550e-01, -5.2761941617684671e-01,
+      4.3270148885021502e-02,  -7.9514379292441917e-01,
+      -6.0487522910864555e-01, 3.6223125868812583e-01,
+      -6.7778068114188961e-01, -6.3984518713517902e-01,
+      5.6126211666505466e-01,  -4.1478314412228895e-01,
+      -7.1619814279896810e-01, 5.0904122200913993e-01,
+      -1.2541675849024123e-01, -8.5155602926950258e-01,
+      3.6219428467870368e-01,  1.8131606454322607e-01,
+      -9.1429742692662042e-01, 1.0156890427467660e-01,
+      4.8792802803496238e-01,  -8.6695443775457692e-01,
+      -2.3428170299690973e-01, 5.0129310187159892e-01,
+      -8.3295696746999992e-01, -4.8469330648994624e-01,
+      3.1081059495555086e-01,  -8.1759964084337722e-01,
+      -6.2894596419916948e-01, -9.3290894095516802e-02,
+      -7.7183144740055520e-01, -5.5003623758867515e-01,
+      -4.1153184032465101e-01, -7.2670604905855885e-01,
+      -1.9749813869025043e-01, -6.0582266347645586e-01,
+      -7.7069668847879413e-01, 1.5341351942535328e-01,
+      -4.6695102814015538e-01, -8.7087371608998432e-01,
+      1.4715200317164617e-01,  -1.8289587007720595e-01,
+      -9.7205729701045585e-01, 4.4291377777147412e-03,
+      1.5894095543133299e-01,  -9.8727815504300565e-01,
+      -2.9936643831865190e-01, 2.9402297485362335e-02,
+      -9.5368508455935652e-01, -2.4141231141133268e-01,
+      -2.9228963418601211e-01, -9.2535769605404206e-01};
+
+  static constexpr std::array<T, 98> weights = {
```

Comment:
Since these are uniformly weighted, we can make this shorter. We'll need to add a small utility to make this `constexpr`, see https://stackoverflow.com/questions/57756557/initializing-a-stdarray-with-a-constant-value 

You can add `create_array` to `include/integratorxx/util/create_array.hpp` and move these over to
```
static constexpr auto weights = create_array<98, T>(4.0 * M_PI / 98.0);
```
etc

Reply:
done